source/application/transforms/babel/index.js

```diff
@@ -0,0 +1,15 @@
+import {transform} from 'babel-core';
+
+export default function createBabelTransform (application) {
+	return async function babelTransform (file, demo) {
+		const source = typeof file.buffer === 'string' ? file.buffer : file.buffer.toString('utf-8');
+		file.buffer = transform(source).code;
```

Comment:
What are the opts here? Are you sure you want to use the babel defaults?
No blacklists or options transforms? No stage configuration?